### PR TITLE
fix: Convert Polish cluster to fast

### LINKED ISSUE #786: Convert Polish Clustering tasks to Fast
Part of #660

### PR DESCRIPTION
**Pull request by isaac-chung**
## Checklist for adding MMTEB dataset

Reason for dataset addition: resolves #786 

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Clustering/pol/PolishClustering.py [hunk 1]
```diff
@@ -21,22 +28,126 @@ class EightTagsClustering(AbsTaskClustering):
...
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
-        n_samples=None,
-        avg_character_length=None,
+        date=("2019-01-01", "2020-05-01"),
+        form=["written"],
+        domains=["Social"],
+        task_subtypes=["Topic classification"],
```
Comment (x-tabdeveloping) [inline mteb/tasks/Clustering/pol/PolishClustering.py:34]: This is topical clustering, no?

**Review by x-tabdeveloping — COMMENTED** [review]
Looks good, the task is not "topical classification" though, is it?

File: mteb/tasks/Clustering/pol/PolishClustering.py (hunk 1 above)
Comment (isaac-chung) [reply mteb/tasks/Clustering/pol/PolishClustering.py:34]: I'm copying that from the paper. We can add thematic clustering too.

Comment (isaac-chung) [reply mteb/tasks/Clustering/pol/PolishClustering.py]: Done!

**Pull request merged**